README.md

```diff
@@ -4,7 +4,8 @@ Laura makes some changes
 
 made some other changes
 
-
```

Comment:
@dominikkawka why was this line removed?

Reply:
Because there was only 1 space in the other sentences